test/common/test_point_type_union_members_access.cpp

```diff
@@ -0,0 +1,216 @@
+  static_assert(pt.data[1] == 3.0f, "");
+  static_assert(pt.data[2] == 4.0f, "");
+  static_assert(pt.data[3] == 1.0f, "");
+  static_assert(pt.data_c[0] == 5.0f, "");
+  static_assert(pt.data_c[1] == 6.0f, "");
+  static_assert(pt.data_c[2] == 7.0f, "");
+  static_assert(pt.data_c[3] == 0.0f, "");
+}*/
+
+
+/*TEST (PointTypeConstruction, PointXYZRGBNormalDefaultConstruction)
+{
+  constexpr const pcl::PointXYZRGBNormal pt;
+  static_assert(pt.data[0] == 0.0f, "");
+  static_assert(pt.data[1] == 0.0f, "");
+  static_assert(pt.data[2] == 0.0f, "");
+  static_assert(pt.data[3] == 1.0f, "");
+  static_assert(pt.data_n[0] == 0.0f, "");
+  static_assert(pt.data_n[1] == 0.0f, "");
+  static_assert(pt.data_n[2] == 0.0f, "");
+  static_assert(pt.data_n[3] == 0.0f, "");
+  static_assert(pt.data_c[0] == 0.0f, "");
+  static_assert(pt.data_c[1] == 0.0f, "");
+  static_assert(pt.data_c[2] == 0.0f, "");
+  static_assert(pt.data_c[3] == 0.0f, "");
+}
+
+TEST (PointTypeConstruction, PointXYZRGBNormalTenScalarsConstruction)
+{
+  constexpr const pcl::PointXYZRGBNormal pt(2.0f, 3.0f, 4.0f, 5u, 6u, 7u, 8.0f, 9.0f, 10.0f, 11.0f);
+  static_assert(pt.data[0] == 2.0f, "");
+  static_assert(pt.data[1] == 3.0f, "");
+  static_assert(pt.data[2] == 4.0f, "");
+  static_assert(pt.data[3] == 1.0f, "");
+  static_assert(pt.data_n[0] == 8.0f, "");
+  static_assert(pt.data_n[1] == 9.0f, "");
+  static_assert(pt.data_n[2] == 10.0f, "");
+  static_assert(pt.data_n[3] == 0.0f, "");
+  static_assert(pt.data_c[0] == 7.0f, "");
+  static_assert(pt.data_c[1] == 6.0f, "");
+  static_assert(pt.data_c[2] == 5.0f, "");
+  static_assert(pt.data_c[3] == 11.0f, "");
+}
+
+TEST (PointTypeConstruction, PointXYZINormalDefaultConstruction)
+{
+  constexpr const pcl::PointXYZINormal pt;
+  static_assert(pt.data[0] == 0.0f, "");
+  static_assert(pt.data[1] == 0.0f, "");
+  static_assert(pt.data[2] == 0.0f, "");
+  static_assert(pt.data[3] == 1.0f, "");
+  static_assert(pt.data_n[0] == 0.0f, "");
+  static_assert(pt.data_n[1] == 0.0f, "");
+  static_assert(pt.data_n[2] == 0.0f, "");
+  static_assert(pt.data_n[3] == 0.0f, "");
+  static_assert(pt.data_c[0] == 0.0f, "");
+  static_assert(pt.data_c[1] == 0.0f, "");
+  static_assert(pt.data_c[2] == 0.0f, "");
+  static_assert(pt.data_c[3] == 0.0f, "");
+}
+
+TEST (PointTypeConstruction, PointXYZINormalTenScalarsConstruction)
+{
+  constexpr const pcl::PointXYZINormal pt(2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f, 9.0f);
+  static_assert(pt.data[0] == 2.0f, "");
+  static_assert(pt.data[1] == 3.0f, "");
+  static_assert(pt.data[2] == 4.0f, "");
+  static_assert(pt.data[3] == 1.0f, "");
+  static_assert(pt.data_n[0] == 6.0f, "");
+  static_assert(pt.data_n[1] == 7.0f, "");
+  static_assert(pt.data_n[2] == 8.0f, "");
+  static_assert(pt.data_n[3] == 0.0f, "");
+  static_assert(pt.data_c[0] == 5.0f, "");
+  static_assert(pt.data_c[1] == 9.0f, "");
+  static_assert(pt.data_c[2] == 0.0f, "");
+  static_assert(pt.data_c[3] == 0.0f, "");
+}*/
```

Comment:
Why are the tests commented?

Reply:
this is not related to this PR, i removed this file